evalml/pipelines/components/transformers/preprocessing/featuretools.py

```diff
@@ -0,0 +1,70 @@
+from featuretools import EntitySet, calculate_feature_matrix, dfs
+
+from evalml.pipelines.components.transformers.transformer import Transformer
+from evalml.utils.gen_utils import (
+    _convert_to_woodwork_structure,
+    _convert_woodwork_types_wrapper
+)
+
+
+class DFSTransformer(Transformer):
```

Comment:
@bchen1116 any reason not to call this `FeaturetoolsTransformer`? I'm not sure people will know what "DFS" means, or will think its a reference to depth-first search 😆

Reply:
Ah that's a fair point. I can change that in the current PR!